### DISCUSSION TITLE
Does wouter have an `generatePath` equivalent?

### DISCUSSION THREAD

**Discussion by earnubs** [molefrog/wouter#308] [General]
Hello,

react-router has `generatePath` https://v5.reactrouter.com/web/api/generatePath to create paths from templates, does `wouter` have an equivalent?

**Comment by perevezencev** [comment]
no?

**Comment by molefrog** [comment]
> no?


 Yes. Wasn't possible in the previous version since we had a custom path parser, but in v3 we switched to `regexparam`. They provide [`inject`](https://github.com/lukeed/regexparam?tab=readme-ov-file#regexparaminjectpattern-string-values-object) method for constructing paths out of params and patterns.

**Reply by perevezencev** [reply]
Do you have any plans to re-export this utility? Or should I install it manually
